go.mod

```diff
@@ -217,6 +217,7 @@ require (
 	github.com/fatih/structs v1.1.0 // indirect
 	github.com/felixge/httpsnoop v1.0.4 // indirect
 	github.com/fsnotify/fsnotify v1.7.0 // indirect
+	github.com/garyburd/go-oauth v0.0.0-20180319155456-bca2e7f09a17 // indirect
```

Comment:
Do you know why that got pulled in? Doesn't look like we use it directly, so it may be due to another dependency having being updated.

Reply:
It was pulled in because I was using the wrong DEP client.